resources/views/people/relationship/edit.blade.php

```diff
@@ -41,7 +41,7 @@
       <div class="pa4-ns ph3 pv2 bb b--gray-monica">
         {{-- This check is for the cultures that are used to say the last name first --}}
         <div class="mb3 mb0-ns">
-          @if (auth()->user()->name_order == 'firstname_lastname')
+          @if (auth()->user()->name_order == 'firstname_lastname_nickname')
```

Comment:
Wath is the user chooses 'firstname_lastname' order? Shouldn't you test 'firstname_lastname_nickname' or  'firstname_lastname' (or 'firstname_nickname_lastname'?) orders ?

Reply:
Yep. Actually all the combinations. I should probably make a function for this.

Reply:
Done